src/main/java/com/mx/iwaconsolti/calculator/Interfaces/ArithmeticOperation.java

```diff
@@ -0,0 +1,5 @@
+package com.mx.iwaconsolti.calculator.Interfaces;
```

Comment:
Is not recommended to create a package specific for interfaces. It is better to put the interface in the same package as the implementations or the implementation as a subdirectory of where the interface is located